[2024-12-23T17:20:06.456+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.transform_data_all_files scheduled__2024-12-23T17:19:00+00:00 [queued]>
[2024-12-23T17:20:06.467+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.transform_data_all_files scheduled__2024-12-23T17:19:00+00:00 [queued]>
[2024-12-23T17:20:06.468+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-12-23T17:20:06.483+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_data_all_files> on 2024-12-23 17:19:00+00:00
[2024-12-23T17:20:06.490+0000] {standard_task_runner.py:60} INFO - Started process 12587 to run task
[2024-12-23T17:20:06.491+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'transform_data_all_files', 'scheduled__2024-12-23T17:19:00+00:00', '--job-id', '996', '--raw', '--subdir', 'DAGS_FOLDER/extract.py', '--cfg-path', '/tmp/tmpdtcnmenv']
[2024-12-23T17:20:06.492+0000] {standard_task_runner.py:88} INFO - Job 996: Subtask transform_data_all_files
[2024-12-23T17:20:06.501+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-12-23T17:20:06.534+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_dag.transform_data_all_files scheduled__2024-12-23T17:19:00+00:00 [running]> on host b853fb0270cd
[2024-12-23T17:20:06.595+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='thiernosidybah232@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='transform_data_all_files' AIRFLOW_CTX_EXECUTION_DATE='2024-12-23T17:19:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-23T17:19:00+00:00'
[2024-12-23T17:20:06.613+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-12-23T17:20:06.620+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=transform_data_all_files, execution_date=20241223T171900, start_date=20241223T172006, end_date=20241223T172006
[2024-12-23T17:20:06.663+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-23T17:20:06.686+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
